src/btree/bt_debug.c

```diff
@@ -1296,20 +1304,20 @@ __debug_cell(WT_DBG *ds, const WT_PAGE_HEADER *dsk, WT_CELL_UNPACK *unpack)
     case WT_CELL_ADDR_INT:
     case WT_CELL_ADDR_LEAF:
     case WT_CELL_ADDR_LEAF_NO:
-        WT_RET(ds->f(ds, ", ts/txn %s,%s/%" PRIu64 ",%s/%" PRIu64,
-          __wt_timestamp_to_string(unpack->newest_durable_ts, ts_string[0]),
-          __wt_timestamp_to_string(unpack->oldest_start_ts, ts_string[1]), unpack->oldest_start_txn,
-          __wt_timestamp_to_string(unpack->newest_stop_ts, ts_string[2]), unpack->newest_stop_txn));
+        WT_RET(ds->f(ds, ", ts/txn %s,%s,%s",
```

Comment:
Remind me: Do we document somewhere what is the order of timestamps / pairs in the output from verify

Reply:
We did when I'd added the `dump_time_pairs` option but this has been removed - along with the documentation. I'll add this to the config for `dump_address` where it will now be printed